testing/checkers_test.go

```diff
@@ -0,0 +1,74 @@
+package testing
+
+import (
+	"testing"
+
+	. "gopkg.in/check.v1"
+)
+
+// Hook up gocheck into the "go test" runner.
+func Test(t *testing.T) { TestingT(t) }
+
+type suite struct{}
+
+var _ = Suite(&suite{})
+
+func (*suite) TestContainsArrayCheck(c *C) {
+	m := []struct {
+		comment  string
+		obtained []string
+		expected []string
+		err      string
+	}{
+		{
+			"simplest",
+			[]string{"a", "b", "c"},
```

Comment:
But no `int`s? `interface2array` implies that you are also interested in arrays of `int`s, not?

Reply:
Well.. I wanted to have `ArrayContains` as general as possible, even if I don't need anything else but `[]string` at the moment. Anyway, I've added a test for int as well now.